composer.json

```diff
@@ -52,13 +52,7 @@
         "tests": [
             "vendor/bin/phpunit --verbose"
         ],
-        "tests-travis": [
-            "XDEBUG_MODE=coverage vendor/bin/phpunit --verbose --configuration phpunit.xml --coverage-clover test/clover.xml"
-        ],
-        "tests-report": [
-            "XDEBUG_MODE=coverage vendor/bin/phpunit --verbose --configuration phpunit.xml --coverage-html test/html-report"
-        ],
-        "phpcs": [
+       "phpcs": [
```

Comment:
Can you check the indention here? My suggestion might have introduced this.

Reply:
I ran https://github.com/ergebnis/composer-normalize

Reply:
Can you please fix the indentation level only?
No need to update the entire file.

Reply:
Fine, I formatted it by hand.